packages/components/card/src/use-card.ts

```diff
@@ -196,6 +195,7 @@ export function useCard(originalProps: UseCardProps) {
     handleClick,
     isFocusVisible,
     getCardProps,
+    getRippelProps: () => ({ripples, onClear: onClearRipple}),
```

Comment:
@li-jia-nan please create the function `getRippleProps` using `useCallback` so we avoid creating it on every render

Reply:
@jrgarciadev  Hello, I have completed the modification, please review again, thank you

Reply:
I totally understand your concern, but I want to know, why we use `useCallback` instead of `useMemo`, like this:

```ts
const getRippleProps = React.useMemo<RippleProps>(
  () => ({ ripples, onClear: onClearRipple }),
  [ripples, onClearRipple],
);
```